File: lint-arwaky/test-project-python/src-python/infrastructure/empty_bypass_adapter.py
# AES016 — empty class inheriting from contract (dead inheritance bypass)
# Placeholder fraud to trick architecture checker
from contract import ILinterAdapterPort

class FakeContractPlaceholder(ILinterAdapterPort):
    """Empty class - dead inheritance bypass."""
    pass

class DummyComplianceStub:
    """Another empty compliance bypass."""
    pass

File: lint-arwaky/test-project-rust/src-rust/infrastructure/empty_bypass_adapter.rs
// This infrastructure file violates AES016 (dead-inheritance-bypass)
// because it defines an empty adapter structure implementing the port contract with no fields or attributes.
use crate::taxonomy::removal_types::RemovalType;
use crate::contract::dummy_port::IDummyPort;

pub struct EmptyBypassAdapter;

impl IDummyPort for EmptyBypassAdapter {
    fn execute_action(&self, input: String) -> bool {
        // empty implementation to bypass checker!
        true
    }
}
